Lib/test/test_monitoring.py

```diff
@@ -1811,14 +1812,14 @@ def test_gh108976(self):
 class TestOptimizer(MonitoringTestBase, unittest.TestCase):
 
     def setUp(self):
-        import _testinternalcapi
+        _testinternalcapi = import_module("_testinternalcapi")
         self.old_opt = _testinternalcapi.get_optimizer()
         opt = _testinternalcapi.new_counter_optimizer()
         _testinternalcapi.set_optimizer(opt)
         super(TestOptimizer, self).setUp()
 
     def tearDown(self):
-        import _testinternalcapi
+        _testinternalcapi = import_module("_testinternalcapi")
```

Comment:
It is not needed, and SkipTest does not work here anyway.

Reply:
True 😆 Addressed in 628896ca12